opm/input/eclipse/Schedule/SummaryState.hpp

```diff
@@ -34,20 +35,22 @@ namespace Opm {
 
 class UDQSet;
 
-// The purpose of this class is to serve as a small container object for
-// computed, ready to use summary values. The values will typically be used
-// by the UDQ, WTEST and ACTIONX calculations. Observe that all value *have
-// been converted to the correct output units*.
+} // namespace Opm
+
+// The purpose of the SummaryState class is to serve as a small container
+// object for computed, ready to use summary values.  The values will
+// typically be used by the UDQ, WTEST and ACTIONX calculations.  Observe
+// that all values are stored in the run's output unit conventions.
 //
 // The main key used to access the content of this container is the eclipse
 // style colon separated string - i.e. 'WWCT:OPX' to get the watercut in
 // well 'OPX'.  The main usage of the SummaryState class is a temporary
 // holding ground while assembling data for the summary output, but it is
 // also used as a context object when evaulating the condition in ACTIONX
-// keywords. For that reason some of the data is duplicated both in the
+// keywords.  For that reason some of the data is duplicated both in the
```

Comment:
hmm

Reply:
> hmm

This was kind of intentional because the rest of the comment also uses two spaces following the full stop.  I've nevertheless reverted that particular aspect of the PR.